### DISCUSSION TITLE
Where can I read Japanese Raws?

### DISCUSSION THREAD

**Discussion by vsatyamesc** [Rattlehead15/tachiyomiOCR#5] [Q&A]
This app is made so that we could read Japanese letters, but if there isn't Japanese letters what should we do?
I can't see any extensions of Manga raw sites. Please add them

**Comment by Rattlehead15** [comment]
Wow just noticed this exists. Actually, Tachiyomi does provide quite a few japanese sources, you just have to install the extensions. If you don't see the extensions for japanese sources, you should tap the top right button, which should show a bunch of switches for different languages. Enable the japanese one and japanese sources should appear. Then you just install the extensions and use them like you would any other tachiyomi source
